model_card_toolkit/core.py

```diff
@@ -404,11 +402,10 @@ def export_format(self,
                          'Call scaffold_assets() to generate model_card.')
 
     # Generate Model Card.
-    jinja_env = jinja2.Environment(
```

Comment:
Personally, I prefer how this was before. I find the previous way easier to read, easier tell what changed when updated, and it leaves more room for longer statements. Which rule covers this?